### PR TITLE
doc: Add separate release details

### LINKED ISSUE #218: Document instructions for package managers in releases
**Summary:** 

Document instructions for package managers in releases.

**Why do we need this?**

We refer to the releases pages in the documentation (Getting Started for example). The instructions to install packages should be there.

**What is already there? What do you see now?**

The downloads of source and binaries, as well as pulling Docker images.

**What is missing? What do you want to see?**

Instructions for package managers. For brew, for example, it should be something like;

```
$ brew tap TheThingsNetwork/lorawan-stack
$ brew install ttn-lw-stack
```

**How do you propose to implement this?**

Not sure; it should be generated though.

**Environment:**

Any

**What can you do yourself and what do you need help with?**

Review

### PR DESCRIPTION
**Pull request by adriansmares**
**Summary:**

Closes #218

This PR adds separate instructions for each operating system based on the currently supported package managers (only `brew`) and manually downloading the binaries.

**Changes:**

- Adds `brew` instructions for MacOS
- Adds manually downloading the binaries instruction for Linux and Windows.

**Notes for Reviewers:**

As requested, the manual installation instructions for the debian packages has not been added yet.

### CODE REVIEW

File: doc/gettingstarted.md [hunk 1]
```diff
@@ -25,16 +25,46 @@ In this guide we will get everything up and running on a server using Docker.
 
 For now, the web interface Console is not available; in this tutorial, we use the command-line interface (CLI) to manage the network.
 
-Get the latest packages or binaries for your operating system from the [releases](https://github.com/TheThingsNetwork/lorawan-stack/releases) page on GitHub.
+Keep in mind that in this tutorial the stack is meant to be run from Docker, and you should run only the CLI commands from the host machine.
```
Comment (johanstokking) [inline doc/gettingstarted.md]: Does "keep in mind" refer to an earlier statement in this doc, and if so, which one?

Also, isn't the point of these CLI commands to _not_ run on the host, like stated here, but to manage from your local machine?

File: doc/gettingstarted.md [hunk 2]
```diff
@@ -25,16 +25,46 @@ In this guide we will get everything up and running on a server using Docker.
...
+
+#### Windows
+
+Currently, only directly downloading the binaries is supported on Windows. 
+
+The CLI can be obtained by downloading the latest `lorawan-stack_windows_amd64.zip` (or `lorawan-stack_windows_386.zip` for x86) from the [releases](https://github.com/TheThingsNetwork/lorawan-stack/releases) page on GitHub. 
+
+After extracting the release into your working directory (such as a separate folder on your Desktop), you can run the CLI commands from the said folder.
+
+#### MacOS
```
Comment (johanstokking) [inline doc/gettingstarted.md]: Casing; `macOS`

**Comment by johanstokking** [comment]
@adriansmares please always close (or reference) an issue.

**Comment by rvolosatovs** [comment]
@johanstokking package managers are not necessarily tied to an OS.
E.g. `brew` or `nix` run on both `darwin` and `linux`

**Comment by johanstokking** [comment]
@rvolosatovs with all the package managers in the world I rather start looking under macOS than scrolling through a list of package managers that may or may not be supported on macOS.

File: doc/gettingstarted.md [hunk 3]
```diff
@@ -25,16 +25,46 @@ In this guide we will get everything up and running on a server using Docker.
 
 For now, the web interface Console is not available; in this tutorial, we use the command-line interface (CLI) to manage the network.
 
-Get the latest packages or binaries for your operating system from the [releases](https://github.com/TheThingsNetwork/lorawan-stack/releases) page on GitHub.
+In this tutorial the stack will be run inside a provided Docker container, and you should run the CLI commands from your local machine.
```
Comment (johanstokking) [inline doc/gettingstarted.md]: `In this tutorial, the stack will run in Docker on the server, and you manage the stack either using the CLI on your local machine or on the server.`

**Pull request merged**